priv/templates/enum.ex.eex

```diff
@@ -2,6 +2,11 @@ defmodule <%= name %> do
   @moduledoc false
   use Protobuf<%= options %>
 
+  def descriptor do
```

Comment:
When the `Credo.Check.Readability.SinglePipe` is enabled in `credo`, each `descriptor` function causes the warning `[R] Use a function call when a pipeline is only one function long`. There are comment options to disable individual `credo` warnings, but these options work only on an entire source file or on a specified number of lines. The variable length of the byte stream to be decoded makes the latter option impractical.

I think it would be better to generate `descriptor` functions for `enum`, `message`, and `service` as follows.
```
def descriptor do
    Elixir.Google.Protobuf.DescriptorProto.decode(<<...>>)
end
```

Reply:
Updated to call the decode function with the binary directly

Reply:
One more `credo` issue. The call to the `decode` function causes the warning `[D] ↘ Nested modules could be aliased at the top of the invoking module.`

- An alias, e.g. `alias Google.Protobuf.DescriptorProto`, could be added to the modules and used in the call to the `decode` function.
- The `credo` comment `# credo:disable-for-next-line Credo.Check.Design.AliasUsage` could be added on the line above the call to the `decode` function.